x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/update.ts

```diff
@@ -81,7 +81,11 @@ export default function createUpdateTests({ getService }: FtrProviderContext) {
                 muteAll: false,
                 mutedInstanceIds: [],
                 scheduledTaskId: createdAlert.scheduledTaskId,
+                createdAt: response.body.createdAt,
+                updatedAt: response.body.updatedAt,
               });
+              expect(Date.parse(response.body.createdAt)).to.be.greaterThan(0);
```

Comment:
I think for the update tests, we should check that the create/update dates:

- aren't the same
- update > create

Can probably just make this one test, something like:

```
expect(Date.parse(response.body.updatedAt)).to.be.greaterThan(Date.parse(response.body.createdAt));
```